crates/config/src/towerfile.rs

```diff
@@ -1,8 +1,8 @@
 use crate::Error;
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
```

Comment:
now we can both read and write towerfiles, we add the Serialize trait to everything that previously had Deserialize